src/java/edu/ucsd/library/dams/file/characterization/model/MetadataConstants.java

```diff
@@ -0,0 +1,52 @@
+package edu.ucsd.library.dams.file.characterization.model;
+
+/**
+ * Terms and constants used in technical metadata
+ * @author lsitu
+ */
+public class MetadataConstants {
+
+    public static final String FILE_NAME = "fileName";
+
+    public static final String FILE_PATH = "filePath";
+
+    public static final String SIZE = "size";
+
+    public static final String DATE_CREATED = "dateCreated";
+
+    public static final String DATE_MODIFIED = "dateModified";
+
+    public static final String FORMAT = "format";
+
+    public static final String VERSION = "version";
+
+    public static final String STATUS = "status";
+
+    public static final String WELL_FORMED = "wellFormed";
+
+    public static final String FITS_VERSION = "fitsVersion";
+
+    public static final String EXIF_VERSION = "exifVersion";
+
+    public static final String DURATION = "duration";
+    
+    public static final String QUALITY = "quality";
+
+    public static final String VALID = "valid";
+
+    public static final String MIME_TYPE = "mimeType";
+
+    public static final String CHECKSUM_CRC32 = "checksumCRC32";
+
+    public static final String CHECKSUM_MD5 = "checksumMD5";
+
+    public static final String CHECKSUM_SHA = "checksumSHA";
```

Comment:
Are there different sha versions available? 256, 512? Or is there only a default?